package.json

```diff
@@ -53,6 +53,9 @@
     "prettier:write": "prettier --write .",
     "test": "jest --verbose",
     "test:cov": "jest --verbose --coverage",
+    "demo:seed": "(cd demo/.strapi-app && pnpm seed:example)",
```

Comment:
Just a random thought, but what do you think about renaming commands linked to the Strapi app as `demo:app:xxx` or `demo:strapi:xxx` instead of `demo:xxx` ?

This is just to anticipate the need to act on the actual demo which could conflict with those commands